Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -0,0 +1,2609 @@
+//
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import QuartzCore
+import ArcGIS
+
+// MARK: - Time Slider Control
+
+public class TimeSlider: UIControl {
+    
+    // MARK: - Enumerations
+    
+    /**
+     Controls how labels on the slider are displayed.
+     */
+    public enum TimeSliderLabelMode {
```

Comment:
@njarecha This could be renamed to `LabelMode` now that it is a subtype of `TimeSlider`.

Reply:
fixed